src/trace-plugin-loader.js

```diff
@@ -116,16 +116,20 @@ function activate(agent) {
       if (!patchSet) {
         // Load the plugin object
         var plugin = originalModuleLoad(instrumentation.file, module, false);
-        patchSet = [];
+        patchSet = {};
```

Comment:
Was `instrumentation.patches[moduleRoot][version]` previously an object or array? If it was an array we should update its documentation as well.